app/jobs/regular/chat_channel_delete.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Jobs
+  class ChatChannelDelete < ::Jobs::Base
+    def execute(args = {})
+      chat_channel = ::ChatChannel.with_deleted.find_by(id: args[:chat_channel_id])
+
+      # this should not really happen, but better to do this than throw an error
+      if chat_channel.blank?
+        Rails.logger.warn("Chat channel #{args[:chat_channel_id]} could not be found, aborting delete job.")
+        return
+      end
+
+      DistributedMutex.synchronize(
+        "delete_chat_channel_#{chat_channel.id}",
+        validity: 1.minute
```

Comment:
Sorry what I meant previously is that we should just follow whatever default that comes with `DistributedMutex` unless we think there is a chance this will hit 1 minute.

Reply:
Oh right, the default is 1m but I don't need to tell it that, will remove